### DISCUSSION TITLE
Read/Write splitting

### DISCUSSION THREAD

**Discussion by billmetangmo** [psycopg/psycopg2#1512] [Q&A]
Hello ,

I want to implement read/write splitting .
Do you know how it can be done using psycopg2 ? 
I found here https://support.huaweicloud.com/intl/en-us/bestpractice-rds/rds_pg_0010.html that this can be done using pg_is_in_recovery but this means that you need to create a new connection.
Is there an easiest solution ?

**Comment by dvarrazzo** [comment]
I think you need different connections, yes
